docs/sources/installation/mac.md

```diff
@@ -1,116 +1,315 @@
-Once you have an initialized virtual machine, you can control it with `boot2docker stop`
-and `boot2docker start`.
+In OS X, the Docker host address is the address of the Linux VM.
+When you start the `boot2docker` process, the VM is assigned an IP address. Under
+`boot2docker` ports on a container map to ports on the VM. To see this in
+practice, work through the exercises on this page.
 
-> **Note:**
-> If you see a message in the terminal that looks something like this:
->
->    `To connect the Docker client to the Docker daemon, please set: export 
-DOCKER_HOST=tcp://192.168.59.103:2375`
-> 
-you can safely set the environment variable as instructed.
 
-View the
-[Boot2Docker ReadMe](https://github.com/boot2docker/boot2docker/blob/master/README.md)
-for more information.
+## Install Boot2Docker
+ 
+1. Go to the [boo2docker/osx-installer ](
+https://github.com/boot2docker/osx-installer/releases/latest) release page.
 
-## Upgrading
+4. Download Boot2Docker by clicking `Boot2Docker-x.x.x.pkg` in the "Downloads"
+section.
 
-1. Download the latest release of the [Docker for OS X Installer](
-   https://github.com/boot2docker/osx-installer/releases/latest)
+3. Install Boot2Docker by double-clicking the package.
 
-2. If Boot2Docker is currently running, stop it with `boot2docker stop`. Then, run
-the installer package, which will update Docker and the Boot2Docker management tool.
+	The installer places Boot2Docker in your "Applications" folder.
 
-3. To complete the upgrade, you also need to update your existing virtual machine. Open a
-terminal window and run:
+The installation places the `docker` and `boot2docker` binaries in your
+`/usr/local/bin` directory.
 
-        $ boot2docker stop
-        $ boot2docker download
-        $ boot2docker start
 
-This will download an .iso image containing a fresh VM and start it up. Your upgrade is
-complete. You can test it by following the directions below.
+## Start the Boot2Docker Application
 
-## Running Docker
+To run a Docker container, you first start the `boot2docker` VM and then issue
+`docker` commands to create, load, and manage containers. You can launch
+`boot2docker` from your Applications folder or from the command line.
 
-{{ include "no-remote-sudo.md" }}
+> **NOTE**: Boot2Docker is designed as a development tool. You should not use
+>  it in production environments. 
 
-From your terminal, you can test that Docker is running with our small `hello-world`
-example image:
-Start the vm (`boot2docker start`) and then run:
+### From the Applications folder
+ 
+When you launch the "Boot2Docker" application from your "Applications" folder, the
+application:
 
-    $ docker run hello-world
+* opens a terminal window
 
-This should download the `hello-world` image, which then creates a small
-container with an executable that prints a brief `Hello from Docker.` message.
+* creates a $HOME/.boot2docker directory
 
-## Container port redirection
+* creates a VirtualBox ISO and certs
```

Comment:
only if it doesn't yet exist - we don't want users to think they'll lose everything if they re-run it, or stop the VM